package.json

```diff
@@ -27,7 +27,7 @@
   "main": "lib/index.js",
   "scripts": {
     "clean": "rimraf lib",
-    "build": "parcel build src/index.ts --out-dir lib/ --target node --no-source-maps",
+    "build": "tsc --noEmit && parcel build src/index.ts --out-dir lib/ --target node --no-source-maps",
```

Comment:
I've noticed that typechecking was not happening during the build process (parcel only compiles TS down to JS but it doesn't typecheck).

`tsc --noEmit` should do the trick

Reply:
Ahh, I didn't catch this. Thanks!